google/cloud/security/inventory/pipelines/load_group_members_pipeline.py

```diff
@@ -127,13 +128,17 @@ def chunker(seq, size):
             return (seq[pos:pos + size] for pos in xrange(0, len(seq), size))
 
         # TODO: keep track of group_ids that are not retrieved/committed to db
-        for group_ids_chunk in chunker(group_ids, GROUP_CHUNK_SIZE):
-            LOGGER.debug('Retrieving a batch of group members')
-            groups_members_map = self._retrieve(group_ids_chunk)
+        chunk_counter = 0
+        for group_ids_in_chunk in chunker(group_ids, self.GROUP_CHUNK_SIZE):
```

Comment:
nit: Fine to leave this as a TODO, but the chunker thing seems generic enough that we could move it to g/c/s/c/gcp_api/common.py, thus letting all the other pipelines use it.